src/main/integrations/sentry-minidump/minidump-loader.ts

```diff
@@ -86,20 +86,35 @@ async function deleteCrashpadMetadataFile(crashesDirectory: string, waitMs: numb
   }
 }
 
+async function readDirsAsync(paths: string[]): Promise<string[]> {
+  const found: string[] = [];
+  for (const path of paths) {
+    try {
+      const files = await readDirAsync(path);
+      found.push(...files.map((file) => join(path, file)));
+    } catch (_) {
+      //
+    }
+  }
+  return found;
+}
+
 function crashpadMinidumpLoader(): MinidumpLoader {
   const crashesDirectory: string = getCrashesDirectory();
   const crashpadSubDirectory = process.platform === 'win32' ? 'reports' : 'completed';
 
+  const dumpDirectories = [join(crashesDirectory, crashpadSubDirectory)];
+
+  if (process.platform === 'darwin') {
+    dumpDirectories.push(join(crashesDirectory, 'pending'));
+  }
+
   return createMinidumpLoader(async () => {
     await deleteCrashpadMetadataFile(crashesDirectory).catch((error) => logger.error(error));
 
-    const dumpDirectory = join(crashesDirectory, crashpadSubDirectory);
-
-    return (await readDirAsync(dumpDirectory))
+    return (await readDirsAsync(dumpDirectories))
```

Comment:
l: Can we extract this out into a variable, little cleaner to understand.

```suggestion
    const loader = await readDirsAsync(dumpDirectories);
    return loader
```